pybamm/__init__.py

```diff
@@ -173,7 +174,7 @@ def version(formatted=False):
 from .parameters.lithium_ion_parameters import LithiumIonParameters
 from .parameters.lead_acid_parameters import LeadAcidParameters
 from .parameters import parameter_sets
-
+from .parameters import mechanical_parameters
```

Comment:
has this been removed now?

Reply:
Yep, I took the suggestion of bundling it to the `lithium_ion_parameters`.